sympy/printing/pretty/tests/test_pretty.py

```diff
@@ -4377,6 +4377,11 @@ def test_gammas():
     assert xpretty(gamma(x), use_unicode=True) == u'Γ(x)'
 
 
+def test_deltas():
+    assert xpretty(DiracDelta(x), use_unicode=True) == u'δ(x)'
+    assert xpretty(DiracDelta(x, 1), use_unicode=True) == u'δ(x, 1)'
```

Comment:
Should this print as:

```
d δ(x)
--------
dx
```

And what should something like `DiracDelta(x - 1)` print as?